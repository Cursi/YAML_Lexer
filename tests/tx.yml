--- # check this
formatting: {from-game: {chat: '(%sender%) %message%', action: '* %sender% %message%'},additional: format} #this
twobytwotable: [ ['a1', 'a2'], ['b1', 'b2'] ]
--- # next doc
combined: [ {key11: 1, key12: 0}, {key21: 0, key22: 1}]
---
simplelist: [e1,e2,1,2,3]
onemorelist: [1,2,3,{x1: 2, y3: 4.5}]
empties: [[],[[],[]],[],[]]
non-empties: [[1,2],[[a,b],['5', '6', '7']],[single],[]]
final: one
