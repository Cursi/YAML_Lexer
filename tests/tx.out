| document: 
| | map value
| | | key: [formatting]
| | | map value
| | | | key: [from-game]
| | | | map value
| | | | | key: [chat]
| | | | | string value: [(%sender%) %message%]
| | | | | key: [action]
| | | | | string value: [* %sender% %message%]
| | | | key: [additional]
| | | | string value: [format]
| | | key: [twobytwotable]
| | | list value
| | | | list value
| | | | | string value: [a1]
| | | | | string value: [a2]
| | | | list value
| | | | | string value: [b1]
| | | | | string value: [b2]
| document: 
| | map value
| | | key: [combined]
| | | list value
| | | | map value
| | | | | key: [key11]
| | | | | numeric value: [1]
| | | | | key: [key12]
| | | | | numeric value: [0]
| | | | map value
| | | | | key: [key21]
| | | | | numeric value: [0]
| | | | | key: [key22]
| | | | | numeric value: [1]
| document: 
| | map value
| | | key: [simplelist]
| | | list value
| | | | string value: [e1]
| | | | string value: [e2]
| | | | numeric value: [1]
| | | | numeric value: [2]
| | | | numeric value: [3]
| | | key: [onemorelist]
| | | list value
| | | | numeric value: [1]
| | | | numeric value: [2]
| | | | numeric value: [3]
| | | | map value
| | | | | key: [x1]
| | | | | numeric value: [2]
| | | | | key: [y3]
| | | | | numeric value: [4.5]
| | | key: [empties]
| | | list value
| | | | list value
| | | | list value
| | | | | list value
| | | | | list value
| | | | list value
| | | | list value
| | | key: [non-empties]
| | | list value
| | | | list value
| | | | | numeric value: [1]
| | | | | numeric value: [2]
| | | | list value
| | | | | list value
| | | | | | string value: [a]
| | | | | | string value: [b]
| | | | | list value
| | | | | | string value: [5]
| | | | | | string value: [6]
| | | | | | string value: [7]
| | | | list value
| | | | | string value: [single]
| | | | list value
| | | key: [final]
| | | string value: [one]